proto/reaction.proto

```diff
@@ -304,11 +333,42 @@ message Vessel {
       CUSTOM = 1;
       NONE = 2;
       OVEN_DRIED = 3;
+      FLAME_DRIED = 4;
+      EVACUATED_BACKFILLED = 5;
+      PURGED = 6;
     }
   }
-  VesselPreparation.VesselPreparationType preparation = 5;
+  repeated VesselPreparation.VesselPreparationType preparation = 5;
   string preparation_details = 6;
-  Volume volume = 7;  // Size (volume) of the vessel.
+  message VesselAttachment {
+    enum VesselAttachmentType {
+      UNSPECIFIED = 0;
+      NONE = 1;
+      CUSTOM = 2;
+      SEPTUM = 3;
+      CAP = 4;
+      REFLUX_CONDENSER = 6;
+      VENT_NEEDLE = 7;
+      DEAN-STARK = 8;
```

Comment:
I don't believe you can use hyphens here. One way to check is to see if the protobuf compiles. If you run `python setup.py install` from the `ord-schema` folder, it will try to compile the protobuf.

Either `DEANSTARK` or `DEAN_STARK` should be fine here probably

Reply:
Done.